AardvarkTests/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.squareup.$(PRODUCT_NAME:rfc1034identifier)</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
its the new recommended build setting in Xcode. Probably not super super useful here, but for watchOS frameworks and applications you have to have this set